Domain/applicationModelInterface.go

```diff
@@ -0,0 +1,16 @@
+package Domain
+
+type UserApplicationModelInterface interface {
+	GetId() int
+	GetPassport() PassportApplicationModelInterface
+	GetJobs() []JobsApplicationModelInterface
+	UserModelInterface
```

Comment:
It is a bad idea to inherit same interface as domain userModel does.
I prefer one another interface, placed in application. Same approach is correct for infrastructure layer also.

Reply:
UserModelInterface isn't a Domain Model, It  is the same part of Interface for application model and view model. Actually may be it's name isn't suitable and I should change it to UserBasicModelInterface